runtime/compiler/compile/J9SymbolReferenceTable.cpp

```diff
@@ -1400,7 +1400,7 @@ J9::SymbolReferenceTable::findOrCreateStaticSymbol(TR::ResolvedMethodSymbol * ow
          TR_OpaqueClassBlock *declaringClass = owningMethod->getDeclaringClassFromFieldOrStatic(comp(), cpIndex);
          if (declaringClass && fej9->isClassInitialized(declaringClass))
             {
-            static const char *dontFoldVarHandle = feGetEnv("TR_DontFoldVarHandle");
+            static const char *foldVarHandleWithoutGuard = feGetEnv("TR_FoldVarHandleWithoutGuard");
```

Comment:
I would prefer we delivered the code and then flipped the sense of this so we only deal with one thing at a time to make narrowing things down easier.

Reply:
Change in this file will be in a different PR.